modules/update-cvo-conditions.adoc

```diff
@@ -0,0 +1,65 @@
++
+Unlike most manifest-reconciliation failures, this error does not immediately result in `Failing=True`. Under some conditions during installs and updates, the CVO will treat this condition as a `Progressing=True` condition and give the operator up to forty minutes to level before reporting `Failing=True`.
+
+RetrievedUpdates:: When `RetrievedUpdates` is set to True, the CVO successfully retrieves updates. When the `RetrievedUpdates` is set to False, the cluster is not able to retrieve recommended updates. When a CVO is unable to retrieve recommended updates the `CannotRetrieveUpdates` alert displays the reason for not retrieving recommended updates.
+
+// When `RetrievedUpdates` is False, reason will be set to explain why, as discussed in the following subsections.
+// In all cases, the impact is that the cluster will not be able to retrieve recommended updates, so cluster admins will need to monitor for available updates on their own or risk falling behind on security or other bugfixes.
+// When CVO is unable to retrieve recommended updates the CannotRetrieveUpdates alert will fire containing the reason.
+// This alert will not fire when the reason updates cannot be retrieved is NoChannel.
+
+NoUpstream:: No upstream server has been set to retrieve updates.
++
+Set `spec.upstream` in `ClusterVersion` to point to a link:https://github.com/openshift/cincinnati/blob/master/docs/design/openshift.md[Cincinnati server] to receive updates.
+
+InvalidURI:: The configured upstream URI is not valid.
++
+Configure the upstream URI by setting `spec.upstream` in `ClusterVersion` to point to a valid link:https://github.com/openshift/cincinnati/blob/master/docs/design/openshift.md[Cincinnati URI].
+
+InvalidID:: The configured clusterID is not a valid UUID.
++
+To configure the clusterID, set `spec.clusterID` to a valid UUID. The UUID should be unique for each cluster. The default value is used report Telemetry and Insights. It may also be used by the CVO when making Cincinnati requests, so that Cincinnati can return update recommendations tailored to the specific cluster.
+
+NoArchitecture:: The set of architectures has not been configured. This is a result of a CVO coding error. There is no mitigation except to upgrade to a new release image with a fixed CVO.
+
+Impact:: The cluster is unable to retrieve recommended updates. This is a result of a CVO coding error. There is no mitigation except to upgrade to a new release image with a fixed CVO.
+
+NoCurrentVersion:: The cluster version does not have a semantic version assigned and is unable to calculate valid upgrades. This is the result of a release-image creation error. There is no mitigation except to upgrade to a new release image with a fixed CVO.
+
+NoChannel:: The update channel is not configured.
++
+To fix, set the channel to a valid value. For example, stable-4.8.
```

Comment:
There are cases where `NoChannel` is a desired state that does not need to be fixed, as discussed in #22252.  We should probably cross-link or otherwise include that channel-clearing discussion here.